File: riak/php/delete/riak_delete.php
#! /usr/bin/php
<?php
// ------------------------------------------------------------------
//	php/delete/riak_delete.php
//
//					Mar/08/2013
//
// ------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include "curl_get.php";
// ------------------------------------------------------------------
print	"*** 開始 ***\n";

$key_in = $argv[1];

print	"key_in = " .$key_in . "\n";

$url_target = 'http://localhost:8098/riak/shimane/' . $key_in;

curl_delete_proc ($url_target);

print	"*** 終了 ***\n";

// ------------------------------------------------------------------
?>

File: riak/python/delete/riak_delete.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	delete/riak_delete.py
#
#					Feb/09/2015
# ----------------------------------------------------------------
import sys
#
sys.path.append ('/var/www/data_base/common/python_common')
from curl_get import curl_delete_proc
# ----------------------------------------------------------------
print ("*** 開始 ***")
#
key_in = sys.argv[1]
print ("%s" % key_in)
#
url_base = 'http://host_ubuntu1:8098/riak/shimane'
url_target = url_base + '/' + key_in
#
curl_delete_proc (url_target)
#
print ("*** 終了 ***")
# ----------------------------------------------------------------
